src/apps/app_muckrock.erl

```diff
@@ -0,0 +1,54 @@
+-module(app_muckrock).
+-export([start/1]).
+-include("ar.hrl").
+
+-define(CHECK_TIME, 60 * 5).
+
+%%% Scrapes and uploads the latest responses received by the Muckrock FOIA
+%%% collection group.
+
+start(WalletFile) ->
+	ssl:start(),
+	Wallet = ar_wallet:load_keyfile(WalletFile),
+	Queue = app_queue:start(Wallet),
+	spawn(fun() -> server(Queue, []) end).
+
+server(Queue, AlreadySeen) ->
+	LatestItems = get_latest(),
+	NewItems = LatestItems -- AlreadySeen,
+	lists:foreach(
+		fun(Item) ->
+			try submit_item(Queue, Item)
+			catch _:_ -> ok end
+		end,
+		NewItems
+	),
+	receive
+		stop -> ok
+	after ?CHECK_TIME * 1000 ->
```

Comment:
Only waiting 1s between every run sounds very little. Or is the muckrock.com changing that often with new PDFs?